generate-docs/action.yml

```diff
@@ -0,0 +1,96 @@
+name: "generate-docs"
+description: "Generate documentation for the project"
+
+inputs:
+  version-name:
+    description: "Version name to use in the documentation"
+    required: true
+  github-access-token:
+    description: "If the repository is private, specify a token."
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        # Fetch with depth=0 to push to gh-pages
+        fetch-depth: 0
+
+    - name: Install MkDocs
+      run: |
+        python3 -m pip install --upgrade pip setuptools wheel
+        pip3 install -r ${GITHUB_ACTION_PATH}/mkdocs-requirements.txt
+      shell: bash
```

Comment:
IMO, writing `shell` between `name` and `run` looks clean.

Reply:
Thanks for your comment. I prefer to follow the official documentation.
https://docs.github.com/en/actions/creating-actions/creating-a-composite-action

Reply:
Okay, but please refrain from resolving conversations before the reviewers check it.
It makes it hard to find the comment.

Reply:
Yes, basically I don't resolve conversations, but in this case I think it would save you some trouble if I resolved the conversation.